meta-luneos/recipes-connectivity/connman/connman-conf/connman.sh.in

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+# Copyright (c) 2019-2022 LG Electronics, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if [ -f /etc/default/connman ] ; then
+    . /etc/default/connman
+fi
+
+set -e
+
+if [ -f @DATADIR@/connman/wired-setup ] ; then
+    . @DATADIR@/connman/wired-setup
+fi
+
+if [ -f /var/lib/connman/debug-mode ] ; then
+    EXTRA_PARAM="-d"
+    export CONNMAN_SUPPLICANT_DEBUG=1
+    export CONNMAN_DHCP_DEBUG=1
+    TEST_CONF="/var/lib/connman/custom-main.conf"
+    if [ -f $TEST_CONF ] ; then
+         EXTRA_PARAM="$EXTRA_PARAM -c $TEST_CONF"
+    fi
+    CUSTOM_CONFIG="/var/lib/connman/custom-wired-config"
+    WIRED_CONFIG="/var/lib/connman/wired.config"
+    if [ -f $CUSTOM_CONFIG ] ; then
+         cp $CUSTOM_CONFIG $WIRED_CONFIG
+    fi
+fi
+
+#
+# Unconditionally activate these network interfaces before starting
+# connmand. Rely on it to deactivate them if they not supposed to stay
+# activated.
+#
+# An example of when an interface should be added to this list is when
+# its hardware (MAC) address changes between the time when it is first
+# detected by the kernel during boot and when it is first activated (say
+# because during initial detection, the driver attempts to read its
+# hardware address from a filesystem that has not yet been mounted).
+#
+# It is assumed that the state into which the kernel places an interface
+# that does RF transmissions allows it to be activated legally in any
+# country before any further configuration of it is done by connmand.
+#
+for iface in @WEBOS_CONNMAN_PREACTIVATE_INTERFACE_LIST@; do
+    ifconfig $iface up || :
+done
+
+# Send DAD(Duplicate Address Detection) packet for link-local address in IPv6
+# there is a timing issue with creating the link-local address and bringing up
+# the eth0 interface to send out the packet.
+# Also We don't use this change in NFS for developer.
+if [ $IS_NFS = "NO" ] ; then
```

Comment:
I remarked an issue with this line, when $IS_NFS is empty, so let's change it to:
if [ "x$IS_NFS" = "xNO" ] ; then